terraform/shared/modules/newrelic/logreview.tf

```diff
@@ -0,0 +1,158 @@
+      height = 3
+
+      nrql_query {
+        query = "FROM Metric SELECT count(*) AS 'Total uploads', average(apm.service.transaction.duration) AS 'Average time (s)' WHERE appName = 'gsa-fac-${var.cf_space_name}' AND path LIKE 'audit.views.upload_report_view:UploadReportView.post'"
+      }
+
+      legend_enabled = true
+    }
+
+    widget_billboard {
+      title = "Global Requests (Non-Upload)"
+
+      row    = 1
+      column = 10
+      width  = 3
+      height = 3
+
+      nrql_query {
+        query = "FROM Metric SELECT count(*) AS 'Total requests', average(apm.service.transaction.duration) * 1000 AS 'Average time (ms)' WHERE appName = 'gsa-fac-${var.cf_space_name}' AND path NOT LIKE '%ExcelFileHandlerView.post' AND path NOT LIKE '%UploadReportView.post'"
+      }
+
+      legend_enabled = true
+    }
+
+    widget_stacked_bar {
+      title = "500 Error Code"
+
+      row    = 3
+      column = 1
+      width  = 12
+      height = 4
+
+      nrql_query {
+        query = "SELECT count(*) FROM Transaction FACET `request.uri` WHERE response.status = '500' and appName = 'gsa-fac-${var.cf_space_name}' SINCE 1 week AGO TIMESERIES"
+      }
+
+      legend_enabled = true
+    }
+
+    widget_stacked_bar {
+      title = "Login Requests"
+
+      row    = 4
+      column = 1
+      width  = 6
+      height = 4
+
+      nrql_query {
+        query = "SELECT count(*) FROM Transaction FACET `request.uri` WHERE request.uri = '/openid/login/' and appName = 'gsa-fac-${var.cf_space_name}' SINCE 1 week AGO TIMESERIES"
+      }
+
+      colors {
+        color = "#00ff04"
+        series_overrides {
+          color       = "#00ff04"
+          series_name = "/openid/login/"
+        }
+      }
+
+      legend_enabled = true
+    }
+
+    widget_stacked_bar {
+      title = "Logout Requests"
+
+      row    = 5
+      column = 1
+      width  = 6
+      height = 4
+
+      nrql_query {
+        query = "SELECT count(*) FROM Transaction FACET `request.uri` WHERE request.uri = '/openid/logout/' and appName = 'gsa-fac-${var.cf_space_name}' SINCE 1 week AGO TIMESERIES"
+      }
+      colors {
+        color = "#ff0000"
+        series_overrides {
+          color       = "#ff0000"
+          series_name = "/openid/logout/"
+        }
+      }
+
+      legend_enabled = true
+    }
+
+    widget_billboard {
+      title = "Infected Files Detected"
+
+      row    = 6
+      column = 1
+      width  = 3
+      height = 3
+
+      nrql_query {
+        query = "SELECT count(*) AS 'Infected Files' FROM Log WHERE tags.space_name = '${var.cf_space_name}' and message LIKE '%ScanResult.INFECTED%'"
+      }
+
+      legend_enabled = true
+    }
+
+    widget_billboard {
+      title = "Django /admin/login/ Count"
+
+      row    = 6
+      column = 4
+      width  = 3
+      height = 3
+
+      nrql_query {
+        query = "SELECT count(`message` as `/admin/login/ hits`) FROM Log WHERE `message` LIKE 'app.fac.gov%/admin/login%' SINCE 7 days ago"

```

Comment:
will this query work in non-prod environments (looking at the url pattern)?

Reply:
It will work, yes, but all dashboard will have it.

Ill modify it a little bit to be each specific environments url.

Reply:
Latest commit should fix this:
local tf plans

space = production 
```
query      = "SELECT count(`message` as `/admin/login/ hits`) FROM Log WHERE `message` LIKE 'app.fac.gov%/admin/login%' SINCE 7 days ago"
```
space = preview
```
"SELECT count(`message` as `/admin/login/ hits`) FROM Log WHERE `message` LIKE 'fac-preview.app.cloud.gov%/admin/login%' SINCE 7 days ago"
```

Not sure how much the lower env admin panels are used, but this should theoretically work for the dashboard and keep it 100% dynamic